pom.xml

```diff
@@ -211,16 +225,28 @@
             <version>RELEASE82</version>
         </dependency>
         <dependency>
-            <groupId>com.github.eirslett</groupId>
-            <artifactId>frontend-maven-plugin</artifactId>
```

Comment:
The frontend dependency was here by accident, right? It is a plugin and it shall be used only in the plugins section.

Reply:
This plugin builds the HTML frontend while building the NetBeans module. Maybe it is the wrong position but this is used while build time.

Reply:
Please check your `pom.xml` - you have the plugin referenced twice. Once in the `plugins` section (correct), once here (incorrect). As a result of listing the plugin among `dependencies` the generated `NBM` file is enormously huge - it contains all the code for this plugin with all its dependencies. This should be cleaned up regardless of this PR.

Reply:
I will have a look thx. But in general I only have the build stuff in the NBM file at the end. Nothing more.

Reply:
node_modules and other development stuff, will not copied over. Those are excluded. I know, that NBM file is more huge, than my previous version, but this comes not from the UI, it must be some maven modules/boilerplate code. I guess.

Reply:
Try it yourself: With and without this dependency and compare the resulting content of the NBM.

Reply:
Ok I will test it. thx :)